examples/nemo/eg3/kernels_trans.py

```diff
@@ -68,104 +68,13 @@
 
 from __future__ import print_function
 from psyclone.psyGen import TransInfo
+from kernel_utils import add_kernels
```

Comment:
I know it is not modified in this PR but could you please change to `./kernels_trans.py` on line 43.

Reply:
Done.